http-gateway/swagger.yaml

```diff
@@ -19,6 +19,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/CloudConfiguration'
+  '/api/v1/oauth/callback':
```

Comment:
I wouldn't document this endpoint. It's not endpoint somebody can query.
What we miss is documentation of ./well-known/ endpoint. Tomas already complained so please exchange it for that one.

Reply:
It is documented: https://github.com/plgd-dev/cloud/blob/v2/http-gateway/swagger.yaml#L8